.github/workflows/github-test.yaml

```diff
@@ -16,12 +16,15 @@ permissions:
 
 defaults:
   run:
-    shell: bash
     working-directory: ./.github
 
 jobs:
   test:
-    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        os: [ubuntu, windows]
+
+    runs-on: ${{ fromJSON('{"ubuntu":"ubuntu-24.04", "windows":"windows-2022"}')[matrix.os] }}
```

Comment:
I always wanted dynamic object literals and hash lookups in devops templating. Useful that you can do this here.